### ISSUE TITLE
How hide breadcrumbs navigation on some docs pages  ?

### ISSUE THREAD

**Issue by Flashpok** [google/docsy#983]
Hi,
Is it possible to hide breadcrumb navigation for the root file of the doc (_index.md) ?

Example : 
I want to display Breadcrumb navigation if the doc (content A) have a child content (content B) : 
content A-> content B
But I want to hide if I visit the main content A page .

thank you,
C.

**Comment by LisaFC** [comment]
@chalin, we should be able to do this, right?

**Comment by chalin** [comment]
Yes, there is some flexibility in the configuration. Is your site publicly available so that I can take a look and have a better idea of what you are trying to achieve?

**Comment by at055612** [comment]
You can hide single level bread crumbs using css (see https://github.com/google/docsy/issues/857#issuecomment-1018442696), but I would agree it would be nice if it was a configurable thing.
